abci/types/types.pb.go

```diff
@@ -1394,7 +1394,7 @@ func _Response_OneofMarshaler(msg proto.Message, b *proto.Buffer) error {
 		}
 	case nil:
 	default:
-		return fmt.Errorf("Response.Value has unexpected type %T", x)
```

Comment:
This file is auto-generated! We should probably change the associated `.proto` file instead.